java/code/src/com/suse/manager/webui/services/SaltServerActionService.java

```diff
@@ -1790,6 +1790,18 @@ else if (state.equals("rebooted") && ((VirtualizationRebootGuestAction)action).i
                     pillar.put("arch", action.getDetails().getArch());
                     pillar.put("cluster_definitions", action.getDetails().getClusterDefinitions());
 
+                    if (action.getDetails().isUefi()) {
+                        Map<String, Object> uefiData = new HashMap<>();
+                        if (action.getDetails().isUefi() && action.getDetails().getUefiLoader() == null) {
```

Comment:
The 1st part of the condition is redundant (already checked on L1793).

Reply:
nice catch! fixed